### PR TITLE
Update terminology template and improve language

### PR DESCRIPTION
**Pull request by ivajloip**

### CODE REVIEW

File: draft-ietf-lpwan-schc-over-lorawan.md [hunk 1]
```diff
@@ -328,24 +330,26 @@ as a part of the ruleId field.
...
 ## Rule ID management
 
 RuleID minimum length MUST be 8 bits, and RECOMMENDED length is 8 bits.
 RuleID MSB is encoded in the LoRaWAN FPort as described in
-{{Fig-lorawan-schc-payload}}. LoRaWAN supports up to 223 application FPorts in
-range [1;223], it implies that RuledID MSB SHOULD be inside this range.
-Application MAY reserve some FPort values for other needs.
+{{lorawan-schc-payload}}. LoRaWAN supports up to 223 application FPorts in
+the range \[1;223\] as defined in section 4.3.2 of {{lora-alliance-spec}}, it implies
+that RuledID MSB SHOULD be inside this range. An application MAY reserve some
```
Comment (jca-klk) [inline draft-ietf-lpwan-schc-over-lorawan.md]: typo 
```suggestion
that RuleID MSB SHOULD be inside this range. An application MAY reserve some
```

File: draft-ietf-lpwan-schc-over-lorawan.md [hunk 2]
```diff
@@ -357,19 +361,20 @@ values are:
...
 control messages of an uplink fragmentation session.
 
-Application can have multiple fragmentation sessions between a device and one
-or several SCHC gateways.  A set of FPort values is REQUIRED for each gateway
+An application can have multiple fragmentation sessions between a device and one
+or several SCHC gateways.  A set of FPort values is REQUIRED for each NGW
 instance the device is required to communicate with.
 
-Mechanism to share those RuleID values is out-of-scope of this document.
+The mechanism for sharing those RuleID values is out-of-scope of this document.
```
Comment (jca-klk) [inline draft-ietf-lpwan-schc-over-lorawan.md]:
```suggestion
The mechanism for sharing those RuleID values is outside the scope of this document.
```

**Review by jca-klk — APPROVED** [review]
+1 👍 
Thanks!

**Comment by ivajloip** [comment]
Thanks @jca-klk for the quick review and the corrections!

File: draft-ietf-lpwan-schc-over-lorawan.md [hunk 3]
```diff
@@ -357,19 +361,20 @@ values are:
...
 
 The only uplink messages using the FPortDown port are the fragmentation SCHC
 control messages of a downlink fragmentation session (ex ACKs). Similarly, the
 only downlink messages using the FPortUp port are the fragmentation SCHC
 control messages of an uplink fragmentation session.
 
-Application can have multiple fragmentation sessions between a device and one
-or several SCHC gateways.  A set of FPort values is REQUIRED for each gateway
+An application can have multiple fragmentation sessions between a device and one
+or several SCHC gateways.  A set of FPort values is REQUIRED for each NGW
```
Comment (Oliv4945) [inline draft-ietf-lpwan-schc-over-lorawan.md]: Good catch, anyway it is not only NGW but NGW + SCHC F/R. Should we put it there or just let NGW so it is easier to read ?

File: draft-ietf-lpwan-schc-over-lorawan.md [hunk 4]
```diff
@@ -413,24 +418,23 @@ field is not used and its size is 0.
...
 SCHC F/R MUST concatenate FPort and LoRaWAN payload to retrieve the SCHC
-fragment as per {{Fig-lorawan-schc-payload}}.
+fragment as per {{lorawan-schc-payload}}.
 
-* SCHC header is two bytes, including FPort byte.
-* **RuleID**: Recommended size is 8 bits in SCHC header.
-* **SCHC fragmentation reliability mode**: `ACK-on-Error`
-* **DTag**: Size is 0 bit, not used
-* **FCN**: The FCN field is encoded on N = 6 bits, so WINDOW_SIZE = 64 tiles
+* SCHC header is two bytes (the FPort byte + 1 additional byte).
```
Comment (Oliv4945) [inline draft-ietf-lpwan-schc-over-lorawan.md]: Should we specify `Minimum SCHC header is two bytes` ? As RuleId can be more than 1 byte

**Review by Oliv4945 — COMMENTED** [review]
Thank you Ivaylo for all those nice changes. I added two comments that are more questions on my side then we are good to go :)

File: draft-ietf-lpwan-schc-over-lorawan.md (hunk 3 above)
Comment (ivajloip) [reply draft-ietf-lpwan-schc-over-lorawan.md]: Thank you @Oliv4945 ! You are right. I think I should have put `SCHC gateway` here even if it repeats the previous sentence. On the other hand, looking at this again I am wondering if the reader can be mislead to think that there could be multiple simultaneous sessions between a device and one SCHC gateway, which would contradict [sec 5.6.1][1] if I am reading it correctly. In practice this text seems to suggests that we can have multiple SCHC datagram exchanges happening simultaneously, which for me could be admissible as long as we use different FPorts as is indicated in this section (I am not sure how practical it's going to be, but I don't see good reason to disallow it). Do you think we should clarify 5.6.1 or you would prefer something else?

[1]: https://acklio.github.io/schc-over-lorawan/ivo-comments-v03/draft-ietf-lpwan-schc-over-lorawan.html#rfc.section.5.6.1

File: draft-ietf-lpwan-schc-over-lorawan.md (hunk 4 above)
Comment (ivajloip) [reply draft-ietf-lpwan-schc-over-lorawan.md]: I agree that maybe we are too restrictive in this case. Please see it my new change resolves your comment and resolve it if it is the case :slightly_smiling_face:

File: draft-ietf-lpwan-schc-over-lorawan.md [hunk 5]
```diff
@@ -371,7 +371,7 @@ only downlink messages using the FPortUp port are the fragmentation SCHC
 control messages of an uplink fragmentation session.
 
 An application can have multiple fragmentation sessions between a device and one
-or several SCHC gateways.  A set of FPort values is REQUIRED for each NGW
+or several SCHC gateways.  A set of FPort values is REQUIRED for each SHCH gateway
```
Comment (Oliv4945) [inline draft-ietf-lpwan-schc-over-lorawan.md:374]: Typo on SCHC

**Review by Oliv4945 — APPROVED** [review]

**Pull request merged**